reflex/components/el/elements/forms.py

```diff
@@ -384,6 +385,33 @@ class Input(BaseHTML):
     # Fired when a key is released
     on_key_up: EventHandler[key_event]
 
+    @classmethod
+    def create(cls, *children, **props):
+        """Create an Input component.
+
+        Args:
+            *children: The children of the component.
+            **props: The properties of the component.
+
+        Returns:
+            The component.
+        """
+        from reflex.vars.number import ternary_operation
+
+        value = props.get("value")
+
+        # React expects an empty string(instead of null) for uncontrolled inputs.
```

Comment:
nit: aren't these controlled inputs when a value is passed

Reply:
ah yeah, good catch